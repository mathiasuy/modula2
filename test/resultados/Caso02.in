listarCDPs
listarPaquetes
listarTramos
ingresarCentro MVD Uruguay Montevideo 1
ingresarCentro MVD YaExiste YaExiste 1
ingresarTramo PDU MVD 100
ingresarTramo MVD PDU 100
ingresarCentro PDU Uruguay Paysandu 3
ingresarTramo MVD PDU 100
ingresarPaquete paq01 ATI MVD 5 [ costo ]
ingresarPaquete paq01 MVD ATI 5 [ costo ]
ingresarCentro STY Uruguay Salto 2
ingresarCentro ATI Uruguay Artigas 3
ingresarTramo PDU STY 50
ingresarTramo STY ATI 80
infoCDP MLZ
infoCDP MVD
ingresarPaquete paq01 MVD ATI 2 [ costo ]
ingresarPaquete paq01 MVD ATI 5 [ costo ]
infoPaquete paq07
infoPaquete paq01
infoCDP MVD
ingresarPaquete paq01 YaExiste YaExiste 5 [ costo ]
ingresarPaquete paq02 MVD ATI 4 [ costo ]
procesarPaquete paq07
procesarPaquete paq01
imprimirRutasOrigDest MVD MLZ 5
imprimirRutasOrigDest MLZ MVD 5
imprimirRutasOrigDest MVD ATI 2
imprimirRutasOrigDest MVD ATI 5
imprimirRutaOpOrigDest MVD MLZ 5
imprimirRutaOpOrigDest MLZ MVD 5
imprimirRutaOpOrigDest MVD ATI 2
imprimirRutaOpOrigDest MVD ATI 5
removerTramo MLZ MVD
removerTramo MVD MLZ
removerTramo MVD ATI
removerTramo PDU STY
procesarPaquete paq01
listarCDPs
listarPaquetes
listarTramos
infoTramo MVD MLZ
infoTramo MLZ MVD
infoTramo MVD ATI
infoTramo MVD PDU
infoCDPActualPaquete paq01
ingresarCentro MLZ Melo Montevideo 6
ingresarTramo PDU MLZ 30
ingresarTramo MLZ STY 40
recalcularRuta paq07
recalcularRuta paq01
infoPaquete paq01
procesarPaquete paq01
infoPaquete paq01
procesarPaquetesCentro NoExiste
procesarPaquetesCentro MLZ
infoPaquete paq01
guardar Caso02.psal
cargar Caso02.pin
listarCDPs
listarPaquetes
infoPaquete paq01
listarTramos
salir
