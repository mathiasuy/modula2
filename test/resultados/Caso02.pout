CENTROS:
5
ATI
Uruguay
Artigas
3
MLZ
Melo
Montevideo
6
MVD
Uruguay
Montevideo
1
PDU
Uruguay
Paysandu
3
STY
Uruguay
Salto
1
TRAMOS:
4
STY
ATI
80
PDU
MLZ
30
MVD
PDU
100
MLZ
STY
40
PAQUETES:
1
paq01
[ costo ]
5
5
4
MVD
PDU
MLZ
STY
ATI
