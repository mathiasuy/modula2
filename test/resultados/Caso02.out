Bienvenidos a InCorreoProg2
version 1.0
InCo-FIng-UdelaR

1> No hay CDPs ingresados en el sistema
2> No existen paquetes en el sistema
3> No existen tramos en el sistema
4> El CDP fue dado de alta
5> Ya existe un CDP con este identificador
6> No existe un CDP con el identificador indicado
7> No existe un CDP con el identificador indicado como destino
8> El CDP fue dado de alta
9> El tramo fue dado de alta
10> No existe un CDP con el identificador indicado
11> No existe un CDP con el identificador indicado como destino
12> El CDP fue dado de alta
13> El CDP fue dado de alta
14> El tramo fue dado de alta
15> El tramo fue dado de alta
16> No existe un CDP con el identificador indicado
17> MVD
Uruguay
Montevideo
1
18> No existe ruta que cumpla las condiciones especificadas
19> El paquete fue dado de alta
20> No existe un paquete con este identificador
21> paq01
[ costo ]
5
4
1
MVD
PDU
STY
ATI
22> MVD
Uruguay
Montevideo
0
23> Ya existe un paquete con este identificador
24> La capacidad del CDP esta agotada
25> No existe un paquete con este identificador
26> El paquete fue procesado
27> No existe un CDP con el identificador indicado como destino
28> No existe un CDP con el identificador indicado
29> No existe ruta que cumpla las condiciones especificadas
30> 
ATI
 STY
  PDU
   MVD
31> No existe un CDP con el identificador indicado como destino
32> No existe un CDP con el identificador indicado
33> No existe ruta que cumpla las condiciones especificadas
34> 4
1
MVD
PDU
STY
ATI
35> No existe un CDP con el identificador indicado
36> No existe un CDP con el identificador indicado como destino
37> No existe un tramo con este origen y destino
38> El tramo fue dado de baja
39> No existe un tramo con este origen y destino
40> ATI MVD PDU STY 
41> paq01
42> MVD-PDU
STY-ATI
43> No existe un CDP con el identificador indicado como destino
44> No existe un CDP con el identificador indicado
45> No existe un tramo con este origen y destino
46> MVD
PDU
100
47> El paquete se encuentra en transito en el centro PDU
48> El CDP fue dado de alta
49> El tramo fue dado de alta
50> El tramo fue dado de alta
51> No existe un paquete con este identificador
52> La ruta ha sido recalculada
53> paq01
[ costo ]
5
5
2
MVD
PDU
MLZ
STY
ATI
54> El paquete fue procesado
55> paq01
[ costo ]
5
5
3
MVD
PDU
MLZ
STY
ATI
56> No existe un CDP con el identificador indicado
57> paq01 - El paquete fue procesado
58> paq01
[ costo ]
5
5
4
MVD
PDU
MLZ
STY
ATI
59> Los datos han sido guardados
60> Los datos han sido cargados
61> ATI MLZ MVD PDU STY 
62> paq01
63> paq01
[ costo ]
5
5
4
MVD
PDU
MLZ
STY
ATI
64> MLZ-STY
MVD-PDU
PDU-MLZ
STY-ATI
65> Fin!